test/Altinn.Notifications.IntegrationTests/Notifications.Persistence/FunctionTests.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Altinn.Notifications.Core.Repository.Interfaces;
+using Altinn.Notifications.IntegrationTests.Utils;
+using Altinn.Notifications.Persistence.Repository;
+
+using Xunit;
+
+namespace Altinn.Notifications.IntegrationTests.Notifications.Persistence
+{
+    public class FunctionTests
+    {
+        /// <summary>
+        /// Scenario: Registered email limit timeout in db  has passed
+        /// Expected side effect: Value is reset to NULL when getemails_statusnew_updatestatus is called by <see cref="EmailNotificationRepository"/>    
+        /// </summary>
+        [Fact]
+        public async Task Run_getemails_statusnew_updatestatus_ConfirmSideEffects()
+        {
+            // Arrange
+            string sql = @"UPDATE notifications.resourcelimitlog
+                        SET emaillimittimeout = '2023-06-16 18:32:29.175852+01'
```

Comment:
Making an update with a local time. Is this persisted or is it automatically converted to UTC time?